ui/app/rif/components/subDomains.js

```diff
@@ -70,14 +69,15 @@ class Subdomains extends Component {
             contentClasses={classes.content}
             actionClasses={classes.contentActions}
             text={subdomain.name}
-            enableRightChevron={true}
+            enableRightChevron={true && !subdomain.action}
```

Comment:
!subdomain.action should be enough

Reply:
You're right! I let the true from previous enabledRightChevron haha